CMakeLists.txt

```diff
@@ -242,7 +242,7 @@ add_subdirectory(dhimpl)
 add_subdirectory(sqliteimpl)
 add_subdirectory(tests)
 add_subdirectory(templates/security)
-add_subdirectory(docs)
+add_subdirectory(docs/api)
```

Comment:
Could not move /src/docs to /docs since /docs already exists. /src/docs are just the config to generate api docs. So /docs/api made sense.